doc-content/enterprise-only/openshift/ref-custom-resource-parameters.adoc

```diff
@@ -27,7 +27,7 @@ When you use the Dashbuilder Container Image within operator, you can configure
 |`env` | -- | Represents an environment variable present in a Container. | --
 |===
 
-You can use Operator to set environment variables by using the `env` property. All configurations listed above with a corresponding environment variable is shown in the following example:
+You can use Operator to set environment variables by using the `env` property. The following example sets the value of the `DASHBUILDER_UPLOAD_SIZE` property to `1000`.
```

Comment:
Can you please check the occurrence of the **Operator** term and change it accordingly? 
Which one is the correct Operator or operator?